failpoint.go

```diff
@@ -56,9 +61,24 @@ func (fp *Failpoint) Enable(inTerms string) error {
 		return err
 	}
 	fp.mu.Lock()
+	defer fp.mu.Unlock()
 	fp.t = t
+	fp.hitCnt = 0
 	fp.waitChan = make(chan struct{})
-	fp.mu.Unlock()
+	return nil
+}
+
```

Comment:
Public repositories/open source repos should have comments for every exported function in the format:

```
// EnableWithValue ... description ...
```

Reply:
Addressed